extensions/cornerstone/src/commandsModule.ts

```
@@ -2615,8 +3168,20 @@ function commandsModule({
     setViewportPreset: {
       commandFn: actions.setViewportPreset,
     },
-    setVolumeRenderingQulaity: {
-      commandFn: actions.setVolumeRenderingQulaity,
+    setSampleDistanceMultiplier: {
+      commandFn: actions.setSampleDistanceMultiplier,
+    },
+    setViewport3DViewDirection: {
+      commandFn: actions.setViewport3DViewDirection,
+    },
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)